moto/iam/responses.py

```diff
@@ -492,9 +492,10 @@ def list_users(self):
         path_prefix = self._get_param("PathPrefix")
         marker = self._get_param("Marker")
         max_items = self._get_param("MaxItems")
+        is_truncated = self._get_bool_param("IsTruncated", False)
```

Comment:
This is not a user supplied argument, but depends on whether we should paginate this call. Considering pagination is not yet implemented for this particular method, this attribute can always be False (for now at least)